challenges/functions.js

```diff
@@ -7,26 +7,40 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+function consume (a, b, cb){
+	return cb(a,b);
+}
 
 /* Step 2: Create several functions to callback with consume();
```

Comment:
*tip* when it says 'return' it should  have a return statement in your function, when the questions asks you to log something the function will have a console.log()